index.html

```diff
@@ -1,11 +1,188 @@
+            <h3 class="features__title">ПЛАНУВАННЯ</h3>
+            <p class="features__text">
+              Так само консультація з широким активом значною мірою зумовлює.
+            </p>
+          </li>
+
+          <li class="features__item">
+            <h3 class="features__title">СУЧАСНІ ТЕХНОЛОГІЇ</h3>
+            <p class="features__text">
+              Значимість цих проблем настільки очевидна, що реалізація
+              планових завдань.
+            </p>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="services">
+      <div class="container">
+        <h2 class="services__title">Чим ми займаємося</h2>
+        <ul class="services__list">
+          <li class="services__item">
+            <img src="./images/services--1.jpg" alt="services--1" class="services__image" width="370" height="294">
+          </li>
 
+          <li class="services__item">
+            <img src="./images/services--2.jpg" alt="services--2" class="services__image" width="370" height="294">
+          </li>
+
+          <li class="services__item">
+            <img src="./images/services--3.jpg" alt="services--3" class="services__image" width="370" height="294">
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="team">
+      <div class="container">
+        <h2 class="team__title">Наша команда</h2>
+
+        <ul class="team__list">
+          <li class="team__item person">
+            <img src="./images/team-1.jpg" alt="team-1" class="person__image" width="270" height="260">
+            <div class="person__info">
+              <h3 class="person__name">Ігор Дем'яненко</h3>
+              <p class="person__role">Product Designer</p>
+            </div>
+          </li>
+
+          <li class="team__item person">
+            <img src="./images/team-2.jpg" alt="team-2" class="person__image" width="270" height="260">
+            <div class="person__info">
+              <h3 class="person__name">Ольга Рєпіна</h3>
+              <p class="person__role">Frontend Developer</p>
+            </div>
+          </li>
+
+          <li class="team__item person">
+            <img src="./images/team-3.jpg" alt="team-3" class="person__image" width="270" height="260">
+            <div class="person__info">
+              <h3 class="person__name">Микола Тарасов</h3>
+              <p class="person__role">Marketing</p>
+            </div>
+          </li>
+
+          <li class="team__item person">
+            <img src="./images/team-4.jpg" alt="team-4" class="person__image" width="270" height="260">
+            <div class="person__info">
+              <h3 class="person__name">Михайло Єрмаков</h3>
+              <p class="person__role">UI Designer</p>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+  </main>
+
+  <footer class="footer">
+    <div class="container">
+      <div class="footer__content">
+        
+        <a href="/" class="logo footer__logo">
```

Comment:
use the relative link to **`./index.html`** here, just as you did in the **`header`**